tests/NLog.UnitTests/Layouts/CsvLayoutTests.cs

```diff
@@ -331,6 +331,7 @@ public void CsvLayoutCachingTest()
                         new CsvColumn("date", "${longdate}"),
                         new CsvColumn("level", "${level}"),
                         new CsvColumn("message", "${message}"),
+                        new CsvColumn("threadid", "${threadid}"),
```

Comment:
Well the unit-test expects the render-result to be cached. And after NLog ver. 4.4.2 then it only caches threadAgnostic = false (Default when not initialized properly)